nemo/src/model/rule_model.rs

```diff
@@ -165,10 +165,7 @@ impl std::fmt::Display for Term {
 impl Term {
     /// Check if the term is ground.
     pub fn is_ground(&self) -> bool {
-        matches!(
-            self,
-            Self::Constant(_) | Self::NumericLiteral(_) | Self::RdfLiteral(_)
-        )
+        !matches!(self, Self::Variable(_))
```

Comment:
I would prefer to have this default to false-negatives for newly-added ground variants rather than having it default to false-positive.